src/libraries/Common/src/Microsoft/Win32/SafeHandles/GssSafeHandles.PlatformNotSupported.cs

```diff
@@ -0,0 +1,58 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System;
+using System.Diagnostics;
+using System.Runtime.InteropServices;
+using System.Text;
+
+namespace Microsoft.Win32.SafeHandles
+{
+    internal sealed class SafeGssNameHandle : SafeHandle
+    {
+        public static SafeGssNameHandle CreateUser(string name) => throw new PlatformNotSupportedException();
+        public static SafeGssNameHandle CreateTarget(string name) => throw new PlatformNotSupportedException();
```

Comment:
Can these two `Create` methods be removed if you exclude `SafeDeleteNegoContext.cs` from .csproj?

Reply:
Done, thanks